src/Aspire.Hosting.Azure.CosmosDB/AzureCosmosDBResource.cs

```diff
@@ -125,7 +125,7 @@ public override void AddRoleAssignments(IAddRoleAssignmentsContext roleAssignmen
         AzureCosmosExtensions.AddContributorRoleAssignment(infra, cosmosAccount, principalId);
     }
 
-    internal ReferenceExpression GetConnectionString(string? databaseName = null, string? containerName = null)
+    internal ReferenceExpression GetConnectionString(string resourceName, string? databaseName = null, string? containerName = null)
```

Comment:
I wonder if we should rename this to `GetChildConnectionString`. And then `resourceName` could be `childResourceName`.